interfaces/opensearch_client/v0/README.md

```diff
@@ -20,6 +20,8 @@ As with all Juju relations, the `opensearch-client` interface consists of two pa
 
 Both the Requirer and the Provider need to adhere to criteria to be considered compatible with the interface.
 
+Passing sensitive information goes via Juju secrets. Corresponding pieces of information are grouped together in a single secret.
```

Comment:
```suggestion
Sensitive information is transmitted through Juju secrets rather than directly through the relation data bag(s). Corresponding pieces of information are grouped together in a single secret.
```